spec/features/responsible_body/submitting_a_spreadsheet_request_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+require 'shared/filling_in_forms'
+
+RSpec.feature 'Submitting a bulk ExtraMobileDataRequest request', type: :feature do
+  context 'not signed in' do
+    it 'does not show the link in the nav' do
+      visit '/'
+      expect(page).not_to have_text('Tell us who needs more data')
+    end
+
+    scenario 'visiting the form directly should redirect to sign_in' do
+      visit new_responsible_body_mobile_bulk_request_path
+      expect(page).to have_current_path(sign_in_path)
+    end
+  end
+
+  context 'signed in' do
+    let(:user) { create(:local_authority_user) }
+    let(:mobile_network) { create(:mobile_network) }
+
+    before do
+      mobile_network
+      sign_in_as user
+      # prevent api call to Notify
+      ActiveJob::Base.queue_adapter = :test
+    end
+
+    after do
+      ActiveJob::Base.queue_adapter = :inline
+    end
```

Comment:
```suggestion
    before do
      mobile_network
      sign_in_as user
    end
  
    around do
      # prevent api call to Notify
      original_queue_adapter = ActiveJob::Base.queue_adapter
      ActiveJob::Base.queue_adapter = :test
      example.run
      ActiveJob::Base.queue_adapter = original_queue_adapter
    end
```

Reply:
Some of this should be addressed by https://github.com/DFE-Digital/get-help-with-tech/pull/214

Reply:
Updated in 9e9eea4